synda/pipeline/clean/deduplicates.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+from sklearn.metrics.pairwise import cosine_similarity
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sqlmodel import Session
+
+from synda.pipeline.executor import Executor
+from synda.model.node import Node
+from synda.progress_manager import ProgressManager
+from synda.model.step import Step
+
+class Deduplicate(Executor):    
```

Comment:
Could you rename the class `DeduplicateTFIDF`?

And the file `deduplicate_td_idf.py`

Reply:
updated!